src/admin-views/events-meta-box.php

```diff
@@ -223,7 +223,7 @@ class="tribe-field-timezone tribe-dropdown hide-if-js"
 							id='EventCurrencySymbol'
 							name='EventCurrencySymbol'
 							size='2'
-							value='<?php echo isset( $_EventCurrencySymbol ) ? esc_attr( $_EventCurrencySymbol ) : tribe_get_option( 'defaultCurrencySymbol', '$' ); ?>'
+							value='<?php echo isset( $_EventCurrencySymbol ) && ! empty( $_EventCurrencySymbol ) ? esc_attr( $_EventCurrencySymbol ) : tribe_get_option( 'defaultCurrencySymbol', '$' ); ?>'
```

Comment:
isset + empty is redundant. empty dosn't throw an error if the variable isn't set.
```suggestion
							value='<?php echo ! empty( $_EventCurrencySymbol ) ? esc_attr( $_EventCurrencySymbol ) : tribe_get_option( 'defaultCurrencySymbol', '$' ); ?>'
```

Reply:
Also, we're not escaping if we get the option? I'd do something like:

```php 
<?php $currency_symbol = $_EventCurrencySymbol ?? tribe_get_option( 'defaultCurrencySymbol', '$' ); ?>
value=<?php echo esc_attr( $currency_symbol ); ?>
```
`null coalesce` simplifies and we ensure the value is always escaped.

Reply:
- Yes but the bug here was exactly because the value was indeed set, but as an empty string `''` .
- I didn't escape the `get_option` because it was already there raw, maybe it is already validated. Ok, will make the change